README.md

```diff
@@ -564,3 +596,40 @@ Nexus instead. The modified files will be accessible at the
 again in a future request, it will use it directly from Nexus rather than downloading it and
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
+
+### pip
+
+The pip package manager works by parsing the `requirements.txt` and `requirements-build.txt` files
+present in the source repository to determine what dependencies are required to build the
+application. It is possible to specify different file path(s) for the requirements files as long
+as the files use the expected format.
+
+Cachito then creates two repositories in an instance of Nexus it manages that contain just the
+dependencies discovered in the requirements files. PyPI dependencies are uploaded to a PyPI hosted
+repository, external dependencies are uploaded to a raw repository. Connection information for the
+hosted repository is provided as the `PIP_INDEX_URL` environment variable accessible at the
+`/api/v1/requests/<id>/environment-variables` endpoint. To make external dependencies available,
+Cachito modifies the requirements files for the request by replacing relevant entries with their
+corresponding URLs from the raw repository. The modified requirements files are accessible at the
+`/api/v1/requests/<id>/configuration-files` endpoint.
```

Comment:
It would be nice to have a note/warning saying that credentials for the temporary repositories will be exposed here. We could also mention this is not an issue, since the reason for them not being open for anonymous access is due to nexus technical limitations.

Reply:
Good idea, added